sql/moz-fx-data-shared-prod/telemetry/cohort_daily_statistics/view.sql

```diff
@@ -2,8 +2,39 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.telemetry.cohort_daily_statistics`
 AS
 SELECT
-  *
+  activity_date,
+  cohort_date,
+  normalized_app_name,
+  normalized_channel,
+  normalized_os,
+  normalized_os_version,
+  attribution_source,
+  attribution_medium,
+  attribution_campaign,
+  attribution_content,
+  attribution_experiment,
+  attribution_variation,
+  num_clients_active_on_day,
+  num_clients_in_cohort
 FROM
   `moz-fx-data-shared-prod.telemetry_derived.cohort_daily_statistics_v1`
 WHERE
   normalized_app_name != "Firefox Desktop"
+UNION ALL
+SELECT
+  submission_date AS activity_date,
+  first_seen_date AS cohort_date,
+  "Firefox Desktop" AS normalized_app_name,
+  normalized_channel,
+  normalized_os,
+  normalized_os_version,
+  attribution_source,
+  attribution_medium,
+  attribution_campaign,
+  attribution_content,
+  attribution_experiment,
+  NULL AS attribution_variation,
```

Comment:
In the absence of other context/type information BigQuery defaults to assuming literal `NULL` expressions are the `INT64` type for some reason.  Here based on the `UNION` and the result schema of the first query BigQuery can tell this should be `STRING`, so this does work as written, but IMO it's best to be explicit.
```suggestion
  CAST(NULL AS STRING) AS attribution_variation,
```